integration-tests/esbuild/build.esm-hono-output-cjs.mjs

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+import ddPlugin from 'dd-trace/esbuild.js'
+import esbuild from 'esbuild'
+import path from 'node:path'
+import { fileURLToPath } from 'node:url'
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url))
+const SCRIPT = path.join(__dirname, 'hono-out.cjs')
+
+const entryPoint = path.join(__dirname, 'hono.mjs')
+const external = [
+  // required if you use native metrics
+  '@datadog/native-metrics',
+
+  // required if you use profiling
+  '@datadog/pprof',
+
+  // required if you use Datadog security features
+  '@datadog/native-appsec',
+  '@datadog/native-iast-taint-tracking',
+  '@datadog/native-iast-rewriter',
+
+  // required if you encounter graphql errors during the build step
+  'graphql/language/visitor',
+  'graphql/language/printer',
+  'graphql/utilities',
+]
+
+esbuild.build({
+  entryPoints: [entryPoint],
+  outfile: SCRIPT,
+  minify: false,
+  bundle: true,
+  platform: 'node',
+  target: 'node22',
+  plugins: [ddPlugin],
+  format: 'cjs',
+  external
+}).catch((err) => {
+  process.exit(1)
```

Comment:
What's the reason for adding this `catch`? If it's not there, I would assume the process would crash with an unhandled rejection error. If it's there in its current form, it will still exit the process with a non-zero exit code, but the error itself will be hidden, which might make debugging harder. But maybe this is run programmatically in which case we just care about the exit code? (same question for all the new `.mjs` files you added in this directory)